polaris-react/src/utilities/features/types.ts

```diff
@@ -1,5 +1,6 @@
 export interface FeaturesConfig {
-  [key: string]: boolean;
+  polaris_summer_editions_2023?: boolean;
```

Comment:
see comment above in the changelog!
```suggestion
  polarisSummerEditions2023?: boolean;
```